recently-edited-content-widget.php

```diff
@@ -108,6 +108,17 @@ protected static function getQuery($args = array())
             $args['meta_value'] = get_current_user_id();
         }
 
+        if (self::$options['since_date']) {
+            $args['date_query'] = array(
+              array(
+                'after' => self::$options['since_date'],
+                'column' => 'post_modified_gmt'
+              )
+            );
+            // Remove page limit when since_date is set
+            $args['posts_per_page'] = -1;
```

Comment:
Please remove.

You're ignoring the `num_items` option without notifying the user.